src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -249,17 +249,21 @@ class RsErrorAnnotator : Annotator, HighlightRangeExtension {
         val mustImplement = canImplement.filterValues { it.isAbstract }
         val implemented = impl.functionList.associateBy { it.name }
 
+        val canImplementTypes = trait.typeAliasList.associateBy { it.name }
+        val mustImplementTypes = canImplementTypes.filterValues { it.typeReference == null }
```

Comment:
@matklad, what do you think of introducing an interface (RsAbstractable) with the `isAbstract` value? We then could get rid of such repentances. Now, `RsTypeAlias` and `RsFunction` (for which we already have `isAbstract`) are good candidates to share this interface.